BaseCollector.py

```diff
@@ -162,3 +167,96 @@ def wait_for_inventory_data(self):
                 print("waiting for initial iteration: " + type(self).__name__)
         print("done: initial query " + type(self).__name__)
         return
+
+    def generate_gauges(self, metric_type, calling_class, vrops_entity_name, labelnames):
+        if not isinstance(labelnames, list):
+            print("Can't generate Gauges without label list, called from", calling_class)
+            return False
+        # switching between metric and property types
+        if metric_type == 'metric':
```

Comment:
Struggled about this naming a little bit. As we usually differentiating between `stats` and `property` I would expect `metric_type` checking  `stats`. The counterpart is checking `property`.

Reply:
Agreed.